Main page tweaks, potential fixes using HTML, missing AUDIT_ log entries (#49)
MoonlightCapital/MoonlightBot-docs: README.md

Welcome to the documentation for MoonlightBot! MoonlightBot is a powerful moderation bot for Discord with a focus on user-friendliness and efficiency. It's simple to use and easy to set up, and is run and maintained entirely by volunteers. This documentation will teach you everything you need to know to use MoonlightBot to its fullest potential.

## Getting Started

First, add the bot to your server using [this invite link](https://discord.com/api/oauth2/authorize?client_id=314110696071888896&permissions=1512298638534&scope=applications.commands%20bot). It's recommended (but not required) that you grant it all permissions it requests, so all its features will work correctly.
First, add MoonlightBot to your server using [this invite link](https://discord.com/api/oauth2/authorize?client_id=314110696071888896&permissions=1512298638534&scope=applications.commands%20bot). It is recommended (but not required) that you grant all requested permissions to ensure all features work correctly.

{% hint style="info" %} You can only add bots to servers in which you have the Manage Server permission. {% endhint %}

> Once you've added it to your server, you can verify that it's working by using the `/ping` command, which simply checks the bot's latency.

Reviewer: Very nice, linking to command pages should be the standard!

If this is your first time using MoonlightBot, you'll receive a Direct Message welcoming you to the bot and providing several recommendations, including reading the [Acceptable Use Policy](./policies/acceptable-use-policy.md). Please read this carefully, as a violation of the Acceptable Use Policy can result in you being banned from using the service.

![Welcome message](./.gitbook/assets/MainPageWelcome.png)

> We also suggest that you review the [Moderation Tutorial](./start-up/moderation-tutorial.md) and share it with your server staff once you've finished configuring MoonlightBot.

Reviewer: ```suggestion
We also suggest that you review the [Moderation Tutorial](./start-up/moderation-tutorial.md) and share it with your server moderators and administrators once you've finished configuring MoonlightBot.
```

I prefer it this way

## Command Permissions

MoonlightBot uses Discord's built-in permissions system to control who is and is not be able to execute certain commands. Some commands have required permissions set by default, and overrides for specific users and roles can be applied to all commands. To set up permissions properly, please follow the [Permission Tutorial](./start-up/permission-tutorial.md).

## Logging

MoonlightBot offers highly-configurable logging, and can log several kinds of actions to one or more channels. To enable and configure logging for a specific channel, use the command
```
/config channels channel:LOG-CHANNEL logs:Open editor
```
where `LOG-CHANNEL` is the channel you want logs posted to.

> An editor will open where you can enter items or categories from the [list of log names](./advanced/list-of-log-names.md), or an asterisk (`*`) to log everything. The list of items and categories to log should be separated by commas and spaces, like so: `BAN, KICK, USER_JOIN, USER_LEFT`

Reviewer: Might be worth adding a screenshot of the form

## Mute Setup

MoonlightBot can mute users both temporarily and permanently. Use the command
```
/create-muterole
```

> to set up a new `@Muted` role with the correct permissions, or to specify an existing role to be used.

Reviewer: The leave blank for existing role is important information I'd keep

Author: I'm worried that specifically saying "leave it blank" implies the `role` option needs to be present, just not filled. The most true-to-function phrasing I can think of is to call further attention that adding the `role` option is used to specify an existing role.

```...to set up a new `@Muted` role with the correct permissions, or to specify an existing role with the `role` option.```

`/create-muterole` on its own, without the `role` option, creates the new role. Only when the `role` option is added does the functionality change. But, if you'd still rather the "leave it blank" phrasing, I will add it.

Reviewer: Can be rephrased as "not specify" or something similar

Author: ```...to set up a mute role. Using the `role` [option] allows you to specify an existing role, or can be left out to create a new `@Muted` role.```

How's this?

Reviewer: `Specifying the \`role\` option allows you to setup it on an existing role, or you can leave it out...`

![Result of /create-muterole command](./.gitbook/assets/MainPageMuterole.png)

> You will now be able to use [`/mute`](./moderation-commands/mute.md),  [`/tempmute`](./moderation-commands/tempmute.md), and [`/unmute`](./moderation-commands/unmute.md).

Reviewer: I suggest adding a paragraph below explaining the evasion ban

```
/config channels channel:LOG-CHANNEL logs:Open editor
```
where `LOG-CHANNEL` is the channel you want logs posted to.

An editor will open where you can enter items or categories from the [list of log names](./advanced/list-of-log-names.md), or an asterisk (`*`) to log everything. The list of items and categories to log should be separated by commas and spaces, like so: `BAN, KICK, USER_JOIN, USER_LEFT`

![Log editor popup](./.gitbook/assets/LogEditor.png)

## Mute Setup

MoonlightBot can mute users both temporarily and permanently. Use the command
```
/create-muterole
```
to set up a mute role. Specifying the `role` [option](./start-up/options.md) allows you to set up an existing role, or you can leave it out to create a new `@Muted` role.

![Result of /create-muterole command](./.gitbook/assets/MainPageMuterole.png)

You will now be able to use [`/mute`](./moderation-commands/mute.md),  [`/tempmute`](./moderation-commands/tempmute.md), and [`/unmute`](./moderation-commands/unmute.md).

## Evasion Bans

> Evasions bans are a fallback moderation feature to ensure muted users cannot abuse improperly configured channels. If a user with your server's mute role sends a message in a channel that hasn't been allowed, they will be banned. To enable evasion bans, use the command

Reviewer: ```suggestion
Evasions bans are a fallback moderation feature to ensure muted users cannot abuse improperly configured permissions. If a user with your server's mute role sends a message in a channel that hasn't been allowed, they will be banned. To enable evasion bans, use the command
```

So what happens is that setting an override to allow it will take priority over deny overrides, this aims at falling back or escalating a mute to a harsher ban.

This also requires a Premium sub, you can insert the vote free trial here!

```
/config settings mute-evasion-ban:True
```

> All channels will now be monitored for new messages sent by muted users. To create an exception to the evasion ban and allow muted users to talk in a channel, use

Reviewer: Ignoring a channel will also ignore all threads made under it. For example, I use this for appeals in my server, good to have some practical notes.

```
/config channels channel:IGNORED-CHANNEL ignore-mute-evasion-ban:True
```
where `IGNORED-CHANNEL` is the channel you want ignored.

It is recommended to set up at least one [logging channel](./README.md#logging) with the `BAN` log enabled to see when an evasion ban is triggered.

![Ban log of an evasion ban](./.gitbook/assets/EvasionBanLog.png)

> ## Changing the Bot's Language

Reviewer: I'm considering moving this section above some others wherever it makes sense, to aid people who speak other languages to have their instruction in their own native language instead.

## Command Permissions

> MoonlightBot uses Discord's built-in permissions system to control who is and is not be able to execute certain commands. Some commands have required permissions set by default, and overrides for specific members and roles can be applied to any and all commands. To set up permissions properly, please follow the [Permission Tutorial](./start-up/permission-tutorial.md).

Reviewer 2: Small grammatical error
```suggestion
MoonlightBot uses Discord's built-in permissions system to control who is and is not able to execute certain commands. Some commands have required permissions set by default, and overrides for specific members and roles can be applied to any and all commands. To set up permissions properly, please follow the [Permission Tutorial](./start-up/permission-tutorial.md).
```

Author: Oops, thank you!

## Evasion Bans

> **Evasion Bans required Premium. [Vote for MoonlightBot](./support/upvote-moonlightbot.md) to receive a renewable free trial of premium!**

Reviewer: An info hint is better for this purpose

Evasions bans are a fallback moderation feature to ensure muted members cannot abuse improperly configured permissions by escalating a mute punishment to a harsher ban. If a user with your server's mute role sends a message in a channel that hasn't been allowed, they will be banned. To enable evasion bans, use the command
```
/config settings mute-evasion-ban:True
```
All channels will now be monitored for new messages sent by muted members. To create an exception to the evasion ban and allow muted members to talk in a channel, use
```
/config channels channel:IGNORED-CHANNEL ignore-mute-evasion-ban:True
```
where `IGNORED-CHANNEL` is the channel you want ignored. Ignoring a channel will also ignore threads made in the channel.

It is recommended to set up at least one [logging channel](./README.md#logging) with the `BAN` log enabled to see when an evasion ban is triggered.

![Ban log of an evasion ban](./.gitbook/assets/EvasionBanLog.png)

> To troubleshoot why the muted member was able to send a message in the unintended channel, check the following:

Reviewer: This is too much for the home page! It could be moved onto the page for the create-muterole command

## Evasion Bans

> {% hint style="info" %} Evasion Bans required Premium. <a href="./support/upvote-moonlightbot.md">Vote for MoonlightBot</a> to receive a renewable free trial of premium! {% endhint %}

Reviewer: As discussed on Discord, this should be reworded heavily

Author: Yea i just didnt know if you had found a different word/phrase for "required" since you mentioned that
